src/Sulu/Bundle/ContactBundle/Resources/public/js/components/documents-tab/main.js

```diff
@@ -35,141 +36,146 @@ define([
 
         initialize: function() {
             this.manager = (this.options.type === 'contact') ? ContactManager : AccountManager;
-            this.form = '#documents-form';
-            this.newSelections = [];
-            this.removedSelections = [];
 
             this.manager.loadOrNew(this.options.id).then(function(data) {
                 this.data = data;
-                this.currentSelection = this.getPropertyFromArrayOfObject(this.data.medias, 'id');
+                this.currentSelection = this.sandbox.util.arrayGetColumn(this.data.medias, 'id');
+                this.bindCustomEvents();
                 this.render();
 
-                if (!!this.data && !!this.data.id) {
+                if (!!this.options && !!this.options.id) {
                     if (this.options.type === 'contact' && WidgetGroups.exists('contact-detail')) {
-                        this.initSidebar('/admin/widget-groups/contact-detail?contact=', this.data.id);
+                        this.initSidebar('/admin/widget-groups/contact-detail?contact=', this.options.id);
```

Comment:
we could avoid the condition block if we just use 
"this.initSidebar('/admin/widget-groups/' + this.options.type + '-detail?' + this.options.type + '=', this.options.id);"